src/components/project.js

```diff
@@ -0,0 +1,198 @@
+import { Header, HeaderLineBelow } from "../styles/Header"
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+import styled from "styled-components"
+import Body from "../styles/Body.js"
+import { SB_ORANGE, SB_NAVY_RGBA } from "@colors"
+
+const CardBG = styled.div`
+  margin-bottom: 20px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  border-radius: 20px;
+  transition: 0.5s;
+  width: 300px;
+  height: 300px;
+  text-align: center;
+  &:hover {
+    transform: translateY(-5px);
+  }
+`
+
+const Card = styled.div`
+  margin-top: -20px;
+  width: 300px;
+  height: 300px;
+  border-radius: 20px;
+  background-color: ${SB_NAVY_RGBA(0.8)};
+`
+
+const CardHeader = styled(Header)`
+  padding-top: 45px;
+  font-size: 1.5em;
+  font-weight: 300;
+  color: ${SB_ORANGE};
+  height: 100px;
+  margin-left: 20px;
+  margin-right: 20px;
+`
+const TagsContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  width: 75%;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  height: 50px;
+`
+
+const Tag = styled.div`
+  font-size: 0.9em;
+  display: inline-block;
+  border-radius: 4px;
+  background-color: rgba(0, 0, 0, 0.35);
+  padding-right: 7px;
+  padding-left: 7px;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  margin-left: 3px;
+  margin-right: 3px;
+  margin-bottom: 6px;
+  text-align: center;
+  color: white;
+`
+
+const CardButton = styled.div`
+  z-index: 999;
+  display: inline-block;
+  border: 1px solid;
+  border-radius: 5px;
+  box-shadow: 0 0 5px -1px rgba(0, 0, 0, 0.2);
+  padding-right: 30px;
+  padding-left: 30px;
+  padding-top: 7px;
+  padding-bottom: 7px;
+  text-align: center;
+  color: white;
+  cursor: pointer;
+  ${CardBG}:hover & {
+    color: ${SB_ORANGE};
+  }
+  margin-top: 60px;
+`
+const ExpandedCard = styled.div`
+  position: fixed;
+  border-radius: 10px;
+  background: white;
+  width: 500px;
+  height: auto;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding-left: 30px;
+  padding-right: 30px;
+  padding-bottom: 30px;
+`
+
+const Overlay = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 99;
+  background-color: rgba(0, 0, 0, 0.7);
+`
+const ExpandedHeader = styled(HeaderLineBelow)`
+  font-size: 1.75em;
+`
+
+const ExpandedClose = styled.div`
+  position: fixed;
+  font-size: 2.5em;
+  top: -10px;
+  right: 5px;
+  padding: 10px;
+  cursor: pointer;
+  color: gray;
+  &:hover {
+    color: ${SB_ORANGE};
+  }
+`
+class Project extends Component {
+  static propTypes = {
+    title: PropTypes.string.isRequired,
+    tags: PropTypes.array.isRequired,
+  }
+
+  constructor() {
+    super()
+
+    this.state = {
+      expandedView: false,
+    }
+  }
+
+  changeView = () => {
+    this.setState(prevState => ({
+      expandedView: !prevState.expandedView,
+    }))
+  }
+
+  openGithub = () => {
+    var win = window.open(this.props.github, "_blank")
+    win.focus()
+  }
+
+  renderTags() {
```

Comment:
Why do you use the fat arrow syntax `() => {}` for some functions and the old function syntax `renderTags()` for others?

Reply:
I prefer the fat arrow, it's a little newer but it matches with the ESLint configuration I'm used to.

Reply:
I only use arrow functions when you need to bind this.

Reply:
Maybe I’m too new to JS but to me the binding `this` behavior seems like the more natural way? Is there a reason to not use then when you don’t need to bind `this`?